### DISCUSSION TITLE
Bulk settings

### DISCUSSION THREAD

**Discussion by jtechwyr** [kimai/kimai#4685] [General]
Hi Kevin, I'm setting up work schedules for over 400 existing users from Monday to Friday and also assigning them the number of vacation days they have. I have to do this manually.
Is there a way in the system to set this up collectively for all users?

**Comment by kevinpapst** [comment]
You can simplify your life by creating SQL statements.

All the settings are in the table `kimai2_user_preferences` .

Names of the values are these:
https://github.com/kimai/kimai/blob/e293d62aa1f7a87bab354c82d54c257959f43f53/src/Entity/UserPreference.php#L35-L44
